### PR TITLE
feat(playground-mod): set payback dusd by dfi & add dusd collateral token

### PR DESCRIPTION
**Pull request by canonbrother**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Add DUSD collateral token
Enable DFI to pay DUSD loan by manually setgov for now

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: src/module.playground/setup/setup.loan.collateral.ts [hunk 1]
```diff
@@ -12,6 +12,11 @@ export class SetupLoanCollateral extends PlaygroundSetup<SetCollateralToken> {
         fixedIntervalPriceId: 'DFI/USD',
         factor: new BigNumber('1')
       },
+      {
+        token: 'DUSD',
+        fixedIntervalPriceId: 'DUSD/USD',
+        factor: new BigNumber('1')
```
Comment (fuxingloh) [inline src/module.playground/setup/setup.loan.collateral.ts:18]:
```suggestion
        factor: new BigNumber('0.99')
```

**Review by fuxingloh — CHANGES_REQUESTED** [review]

File: src/module.playground/setup/setup.loan.collateral.ts (hunk 1 above)
Comment (fuxingloh) [reply src/module.playground/setup/setup.loan.collateral.ts:18]: I believe we need to set it to 0.99

File: src/module.playground/setup/setup.gov.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    if (key === 'ATTRIBUTES') {
+      const dusdInfo = await this.client.token.getToken('DUSD')
+      const dusdId = Object.keys(dusdInfo)[0]
+      /* eslint-disable-next-line */
+      var re = new RegExp(':dusdId', 'g');
+      const k = Object.keys(each[key])[0]
+      each[key] = { [k.replace(re, dusdId)]: 'true' }
+    }
+
+    await this.client.masternode.setGov(each)
```
Comment (fuxingloh) [inline src/module.playground/setup/setup.gov.ts:23]: This is not as clean but I think will do for now.

Comment (canonbrother) [reply src/module.playground/setup/setup.gov.ts:23]: :pray:

**Comment by fuxingloh** [comment]
CI / Docker Sanity Test is failing, could you investigate?

**Comment by canonbrother** [comment]
> CI / Docker Sanity Test is failing, could you investigate?

yea sure

File: .github/workflows/ci/docker-compose.yml [hunk 3]
```diff
@@ -42,6 +42,8 @@ services:
       -eunosheight=6
       -eunospayaheight=7
       -fortcanningheight=8
+      -fortcanningmuseumheight=9
+      -fortcanninghillheight=10
```
Comment (fuxingloh) [inline .github/workflows/ci/docker-compose.yml:46]: Oh right, can you also update this for whale, wallet, and scan repos? Thanks!

**Review by fuxingloh — APPROVED** [review]

File: .github/workflows/ci/docker-compose.yml (hunk 3 above)
Comment (canonbrother) [reply .github/workflows/ci/docker-compose.yml:46]: https://github.com/DeFiCh/scan/pull/885/files
https://github.com/DeFiCh/wallet/pull/2062/files
https://github.com/DeFiCh/wallet/pull/2062/files

**Pull request merged**